packages/fuels-code-gen/src/program_bindings/utils.rs

```diff
@@ -155,3 +155,13 @@ pub(crate) fn sdk_provided_custom_types_lookup() -> HashMap<TypePath, TypePath>
     })
     .collect()
 }
+
+pub(crate) fn find_into_impl_type(ttype: &str) -> Option<TokenStream> {
```

Comment:
It might be more readable to step down the generality, and make the name more specific until we need it to be more general.

E.g. `get_equivalent_bech32_type` 

If we ever `Into` something else we can revert.